src/CTA.Rules.Actions/ClassActions.cs

```diff
@@ -425,6 +428,93 @@ ClassDeclarationSyntax AddParametersToMethod(SyntaxGenerator syntaxGenerator, Cl
             return AddParametersToMethod;
         }
 
+        public Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> GetReplaceMvcControllerMethodsBodyAction(string expression)
+        {
+            ClassDeclarationSyntax ReplaceMethodModifiers(SyntaxGenerator syntaxGenerator, ClassDeclarationSyntax node)
+            {
+                var allMembers = node.Members.ToList();
+                var allMethods = allMembers.OfType<MethodDeclarationSyntax>().Where(m => m.Modifiers.Any(mod => mod.Text == Constants.Public));
+
+                while(allMethods != null && allMethods.Count() > 0)
+                {
+                    var method = allMethods.FirstOrDefault();
+                    //ActionResult is a catch all return type
+                    MethodDeclarationActions methodActions = new MethodDeclarationActions();
+                    var addCommentActionFunc = methodActions.GetAddCommentAction(Constants.MonolithServiceComment);
```

Comment:
Do we need to add a comment? Maybe if a comment is needed, this should be a separate action. To be discussed.

Reply:
Also, can we remove the "added by CTA" part

Reply:
Removed as discussed. Using new way to store unique method information.